controller/controller.js

```diff
@@ -0,0 +1,7 @@
+const model = require("../model/model");
+
+exports.getAllTopics = (req, res) => {
+  model.getAllTopics().then((response) => {
```

Comment:
You've named your model and controller the exact same thing - because you aren't destructuring you can see which is the controller and model, but it's a good convention to name your models something similar but different. My go-to is to name my model after the sql request, i.e. selectAllTopics()